docs/appkit/flutter/core/installation.mdx

```diff
@@ -188,28 +188,206 @@ Coinbase Wallet is enabled by default even if, in order to function properly, a
 ### Phantom Wallet support
 
 :::caution important note
-Phantom Wallet support is available from `reown_appkit: ^1.4.0-beta02` and is currently in beta. Any feedback is appreciated.
+Phantom Wallet support is available from `reown_appkit: ^1.4.0-beta03` and is currently in beta. Any feedback is appreciated.
 :::
 
 Phantom Wallet does not use the WalletConnect protocol for communication between the dApp and the wallet, instead it provides an internal API mechanism based on deep/universal links.
 
 This means that pairing topic, session topic, session events and other session-related features are not available when connecting to Phantom Wallet and the interaction is really basic. Dapp sends a request, Phantom Wallet responds. That's all.
 
-Furthermore, Phantom Wallet  deep/universal links mechanism supports interaction only with Solana blockchain.
+Furthermore, Phantom Wallet deep/universal links mechanism supports interaction only with Solana netowkr, meaning that if you have EVM networks configured in your AppKit's instance these are not going to be available to use after connecting with Phantom.
```

Comment:
@quetool found a minimal grammar error: netowkr

can you run `pnpm start` locally to find more of this issues ?